schemas/Quiz.yaml

```diff
@@ -0,0 +1,17 @@
+title: Quiz
+type: object
+required:
+  - list of questions
+properties:
+  Questions:
+    description: Questions that makeup the Quiz
+    type: list
+    Question:
+      description: Phrase that demands a correct answer to
+      type: string
+      Answers:
+        description: Group of responses to a question
+        type: string
+      Correct Answer:
```

Comment:
same as above. a property name can't be separate words.
also all through the document, you are starting property names with capital letters instead of small letters

Reply:
Sorry for that. I tried referring from the User.yaml schema